README.md

```diff
@@ -51,13 +58,30 @@ Environment variables used to control how tests are executed:
   * `TEST_NEO4J_PASS`  
     Password used to connect to Neo4j server.
     Defaults to 'pass'
+  * `TEST_NEO4J_SCHEME`  
+    Scheme to build the URI when contacting the Neo4j server, default "bolt".
+  * `TEST_NEO4J_HTTP_PORT`  
+    The port on which the Neo4j server is listening for HTTP requests. This is
+    just used to make sure the driver does not try to speak bolt over this port.
+  * `TEST_NEO4J_VERSION`  
```

Comment:
(again for another PR) Go integration tests are configured in a similar and I'm wondering if it'd make sense to just dynamically retrieve the version instead?

Reply:
I think I prefer explicitly stating the server version. Otherwise, you have to rely on the driver to tell you what server version you are testing against. I'd rather rely on the driver to test the driver, if you understand.